Client.Test/ItNotificationEndpointsApiTest.cs

```diff
@@ -29,7 +29,7 @@ public new async Task SetUp()
         [Test]
         public async Task CreateSlackEndpoint()
         {
-            var now = DateTime.UtcNow;
+            var now = DateTime.UtcNow.AddMilliseconds(-10);
```

Comment:
I'm not happy with these workarounds but when running on Windows these tests fail...
![image](https://github.com/influxdata/influxdb-client-csharp/assets/22419043/2b67db9c-bfba-4a18-a913-61592fdaf3dc)

Reply:
From my POV, it's ok. The management API is little bit tricky ;)